addons/project/views/project_project_views.xml

```diff
@@ -85,31 +85,34 @@
                         <page name="settings" string="Settings">
                             <group>
                                 <group>
-                                    <field name="privacy_visibility" widget="radio"/>
-                                    <span colspan="2" class="text-muted o_row ps-1" invisible="access_instruction_message == ''">
-                                        <i class="fa fa-lightbulb-o"/>&amp;nbsp;<field class="d-inline" name="access_instruction_message" nolabel="1"/>
-                                    </span>
-                                    <span colspan="2" class="text-muted o_row ps-1" invisible="privacy_visibility_warning == ''">
-                                        <i class="fa fa-warning"/>&amp;nbsp;<field class="d-inline" name="privacy_visibility_warning" nolabel="1"/>
-                                    </span>
-                                </group>
-                                <group>
-                                    <div name="alias_def" colspan="2" class="pb-2">
+                                    <div name="alias_def" colspan="2">
                                         <!-- Always display the whole alias in edit mode. It depends in read only -->
                                         <!-- Need to add alias_id in view for getting alias_domain_id by default -->
                                         <field name="alias_id" invisible="1"/>
-                                        <label for="alias_name" class="fw-bold o_form_label" string="Create tasks by sending an email to"/>
-                                        <field name="alias_email" class="oe_read_only d-inline" widget="email" readonly="1" invisible="not alias_name" />
```

Comment:
you have removed this and thus now the alias email is not visible to non-admin users